launch/mapper_pensa.launch

```diff
@@ -1,20 +1,20 @@
 <launch>
```

Comment:
Note: there will be an accompanying PR in `pensa-ros` to accomodate with updated variable names in the launch file